config/key_report_mappings/mappings.py

```diff
@@ -16,6 +19,7 @@
 MAPPINGS = (
     COF_R2_KEY_REPORT_MAPPING,
     COF_R3W2_KEY_REPORT_MAPPING,
+    COF_KEY_REPORT_MAPPING,
```

Comment:
Is this supposed to be?
```suggestion
    COF_R3W3_KEY_REPORT_MAPPING,
```

Reply:
No, i made it generic like the forms. Since the mappings haven't changed between rounds we want to start planning ahead so when the mappings don't change we need minimal code changes